.ci/travis/travis.sh

```diff
@@ -78,6 +78,7 @@ javac8)
         --exclude='InputCustomImportOrderNoPackage2.java' \
         --exclude='InputIllegalTypePackageClassName.java' \
         --exclude='InputVisibilityModifierPackageClassName.java' \
+        --exclude='InputJavaParserSwitchExpressionTest.java' \
```

Comment:
Please make this test input excluded via comment
`//non-compiled`

see https://github.com/checkstyle/checkstyle/blob/master/src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java9/InputJava9TryWithResources.java#L1
for reference

Reply:
:+1: